### PR TITLE
Adds hailer gas (face) masks.

### PR DESCRIPTION
**Pull request by SplinterGP**
Still tweaking a few things but the masks itself work fine, just tweakin the code on turning the mask to and back.(help is appreciated)

### CODE REVIEW

**Comment by SplinterGP** [comment]
added them to HOS and Warden lockers, as well as ERT, dind't test the ERT but i think it fits, the HOS and Warden one i needed to increase size to fit it without removing the face mask. still need to make changelog and a desc info for face masks.

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ 
+/obj/item/clothing/mask/gas/sechailer/verb/halt()
+	set name = "HALT!"
+	set category = "Objects"
+	set desc = "Activate your face mask hailer."
+	var/key = phrase_list[phrase]
+	var/message = phrase_list[key]
+
+	if(cooldown < world.time - 35) // A cooldown, to stop people being jerks
+		if(!safety)
```
Comment (Atermonera) [inline code/modules/clothing/masks/hailer.dm:156]: `safety` is only touched by emag_act, so message should probably just be set there.

**Comment by Atermonera** [comment]
Closing/reopening to reboot travis

**Comment by SplinterGP** [comment]
forgot to check this PR, if needed i can correct those.

**Pull request merged**